### PR TITLE
Get master passing

### PR DESCRIPTION
**Pull request by garrettjstevens**
Trying to get master passing CI so I have a good base for my branch.

### CODE REVIEW

File: packages/jbrowse-web/src/plugins/BigWigAdapter/util.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export function rectifyStats(s) {
 export function calcRealStats(region, features) {
   const { start, end } = region
   const scores = []
-  const feats = features.sort((a, b) => a.start < b.start)
+  const feats = features.sort((a, b) => a.start - b.start)
```
Comment (garrettjstevens) [inline packages/jbrowse-web/src/plugins/BigWigAdapter/util.js]: @cmdcolin This test wasn't passing, but this change seems to fix it. Just wanted to make sure this is still doing what you intended.

File: packages/jbrowse-web/src/plugins/HierarchicalTrackSelectorDrawerWidget/components/Category.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import Typography from '@material-ui/core/Typography'
 import { observer, PropTypes as MobxPropTypes } from 'mobx-react'
 import propTypes from 'prop-types'
 import React from 'react'
+// eslint-disable-next-line import/no-cycle
```
Comment (garrettjstevens) [inline packages/jbrowse-web/src/plugins/HierarchicalTrackSelectorDrawerWidget/components/Category.js]: @rbuels Is there a reason you didn't disable the import/no-cycle with the other eslint fixes? I think it should be fine, but let me know if there's something about the component structure you think needs to be fixed.

File: packages/jbrowse-web/src/plugins/BigWigAdapter/util.js (hunk 1 above)
Comment (cmdcolin) [reply packages/jbrowse-web/src/plugins/BigWigAdapter/util.js]: that is definitely the proper way to sort numbers, looks good!

File: packages/jbrowse-web/src/plugins/HierarchicalTrackSelectorDrawerWidget/components/Category.js (hunk 2 above)
Comment (rbuels) [reply packages/jbrowse-web/src/plugins/HierarchicalTrackSelectorDrawerWidget/components/Category.js]: Was just to prompt you to have a look at those cycles when you got around to it. :-)

**Pull request merged**